src/components/ResultListItem.tsx

```diff
@@ -131,7 +127,7 @@ const ResultListItem: FC<ResultListItemProps> = ({ resource }) => {
         <div>
           <StyledHeader>
             <StyledMaxTwoLinesTypography variant="h4">
-              <StyledLink href={`/resource/${resource.identifier}`}>{resource.features.dlr_title}</StyledLink>
+              <Link href={`/resource/${resource.identifier}`}>{resource.features.dlr_title}</Link>
```

Comment:
litt usikker på om tittelen i trefflista også bør være i standard lenke-farge, eller beholdes som sort. Det stemmer ikke med skissene, men skissene sier heller ingenting om hva som er lenker og ikke. Det blir iallefall litt ekstra tydelig hva man skal trykke på for å se posten.